experiment/euler_gravity/gravitywave.jl

```diff
@@ -1,5 +1,6 @@
 using Atum
 using Atum.EulerGravity
+using Bennu: fieldarray
```

Comment:
I don't think these `using` statements are needed because `Atum` reexports all of Bennu.